src/server/server_state.h

```diff
@@ -4,6 +4,7 @@
 
 #pragma once
 
+#include <boost/circular_buffer.hpp>
```

Comment:
no need.

Reply:
fixed in https://github.com/dragonflydb/dragonfly/pull/1956/commits/0f7f07da8fbed1d4db01f489e2d06eb79defaffc